pkg/pipeline/pipeline_mock.go

```diff
@@ -37,14 +38,14 @@ func (p *Pipeline) Start() error {
 
 func (p *Pipeline) AddOutput(url string) error {
 	if !p.isStream {
-		return ErrCannotAddToFile
+		return errors.New("cannot add rtmp output to file recording")
 	}
 	return nil
 }
 
 func (p *Pipeline) RemoveOutput(url string) error {
 	if !p.isStream {
-		return ErrCannotRemoveFromFile
+		return errors.New("cannot remove rtmp output from file recording")
```

Comment:
curious on why use dynamic errors vs const ones?

Reply:
I moved `errors.go` under the `!test` build constraint to import `*gst.GError`